pkg/cmd/server/origin/ensure.go

```diff
@@ -26,12 +26,9 @@ import (
 // ensureOpenShiftSharedResourcesNamespace is called as part of global policy initialization to ensure shared namespace exists
 func (c *MasterConfig) ensureOpenShiftSharedResourcesNamespace() {
 	if _, err := c.KubeClient().Namespaces().Get(c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace); kapierror.IsNotFound(err) {
-		namespace := &kapi.Namespace{
-			ObjectMeta: kapi.ObjectMeta{Name: c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace},
-		}
-		_, err = c.KubeClient().Namespaces().Create(namespace)
-		if err != nil {
-			glog.Errorf("Error creating namespace: %v due to %v\n", namespace, err)
+		namespace, createErr := c.KubeClient().Namespaces().Create(&kapi.Namespace{ObjectMeta: kapi.ObjectMeta{Name: c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace}})
```

Comment:
we needed to pass the created namespace to `ensureNamespaceServiceAccountRoleBindings` or the namespace annotation update would fail with a missing resource version